### PR TITLE
Add additional related "subtags" upon adding new tags

### LINKED ISSUE #9398: Add additional related "subtags" upon adding new tags
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [app/models/node.rb](https://github.com/publiclab/plots2/blob/main/app/models/node.rb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -850,6 +850,15 @@ def add_tag(tagname, user)
             # add sub-tags:
             subtags = {}
             subtags['pm'] = 'particulate-matter'
+            subtags['pm'] = 'air-quality'
+            subtags['h2s'] = 'hydrogen-sulfide'
+            subtags['near-infrared-camera'] = 'multispectral-imaging'
+            subtags['infragram'] = 'multispectral-imaging'
+            subtags['odors'] = 'odor'
+            subtags['oil'] = 'oil-and-gas'
+            subtags['purple-air'] = 'purpleair'
+            subtags['reagent'] = 'reagents'
+            subtags['spectrometer'] = 'spectrometry'
             if subtags.include?(key)
               add_tag(subtags[key], user)
             end
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:
1. If you are not familiar with GitHub or the pull request model, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.
2. If you are familiar with the terminal or would like to learn to use it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

3. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by aliciapaz**
[Fixes #9398](https://github.com/publiclab/plots2/issues/9398)

Added more subtags in node.rb. 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by aliciapaz** [comment]
I'm sorry I made so many commits to change just one file...I followed the git-flow on [this](https://publiclab.org/notes/warren/11-22-2017/use-git-and-github-to-contribute-and-improve-public-lab-software) article, but for some reason, other files were modified during the process of solving the conflicts so I had to edit them back to the way they are in the upstream main.
I was expecting that changing these files back would make the tests pass, but still no luck :P 
@jywarren, @cesswairimu  I would appreciate some help to improve this PR , please. 
Thank you!

File: app/models/node.rb [hunk 1]
```diff
@@ -850,6 +850,15 @@ def add_tag(tagname, user)
             # add sub-tags:
             subtags = {}
             subtags['pm'] = 'particulate-matter'
+            subtags['pm'] = 'air-quality'
```
Comment (cwils021) [inline app/models/node.rb:853]: this key is what is causing unit test to fail

**Comment by jywarren** [comment]
OK, closing and all good now!!!

File: app/models/node.rb (hunk 1 above)
Comment (jywarren) [reply app/models/node.rb:853]: Yes, great catch!!

```suggestion
```

Here, i've made this change as a "suggestion" which @aliciapaz can accept!

**Comment by aliciapaz** [comment]
Thank you @jywarren and @cwils021 for the help. I'm happy to see that everything works now 🎉 🌞

**Review by Tlazypanda — APPROVED** [review]
@aliciapaz LGTM :tada:

**Comment by jywarren** [comment]
Awesome, thank you @aliciapaz and thank you @Tlazypanda !!!!

**Pull request merged**